RequestManagementSystem/Client/Operation.py

```diff
@@ -201,12 +201,12 @@ def __len__( self ):
   @property
   def sourceSEList( self ):
     """ helper property returning source SEs as a list"""
-    return self.SourceSE.split( "," )
+    return self.SourceSE.split( "," ) if self.SourceSE else ['']
```

Comment:
Are you intentionally putting an empty string in the list?
[''] does not evaluate to false.